src/docs/sdk/performance/span-operations.mdx

```diff
@@ -181,6 +181,7 @@ Serverless related spans are expected to follow OpenTelemetry's [Function as a S
 |            | app.start.cold            |                               |
 | ui         |                           | An operation on a mobile UI   |
 |            | ui.load                   |                               |
+|            | ui.ttid                   |                               |
```

Comment:
Should that be part of `app.start`? or every screen would have its own `ttid`?

Reply:
every screen it's going to have its own `ttid`
we'll also add the `ttfd` (time to full display) to every screen, too, but that's for another pr

Reply:
I think we should mention what ttid stands for = time to initial display. Maybe we could also call it
`ui.load.inital` `ui.load.full`? Most developers won't understand what ttid stands for and they have to go to our docs.

Reply:
could be even `ui.load.initial_draw` and `ui.load.full_draw`. But I agree `ttid` is probably not well-known, so I'd also vote for changing it

Reply:
That would be semantically wrong I guess, `initial_draw` would mean it's drawn but that refers only to the very first frame only.
`full_draw` would mean it's fully drawn, but it's not only about drawing but rather waiting for async content and redrawing it.
so maybe `initial_display` and `full_dislay`? it's pretty much the wording for both acronyms

Reply:
so let's go with `ui.load.initial_display` and `ui.load.full_display`? These make sense to me

Reply:
@philipphofmann does it make sense to you on ios?

Reply:
Yes, that makes sense. I'm happier now with this name as it's self-explanatory.